### PR TITLE
Added automated builds for maintenance branches.

### PR DESCRIPTION
**Pull request by mikeller**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -30,13 +30,14 @@ trigger:
...
+    - "*-maintenance"
 
 pr:
   drafts: false
   branches:
     include:
-      - master
-      - "*-maintenance"
+    - master
+    - "*-maintenance"
```
Comment (McGiverGim) [inline azure-pipelines.yml:40]: I know this is working, but the correct way is indenting with two spaces the list inside the group? At least is how I used it until now but I'm not a Yaml expert...

**Review by McGiverGim — APPROVED** [review]

File: azure-pipelines.yml (hunk 1 above)
Comment (mikeller) [reply azure-pipelines.yml:40]: Me neither - the rest of `azure-pipelines.yml` seems to be placing list elements on the same level as the list name, so this is just making the files consistent.
If it is wrong it should be amended for the entire file in a separate pull request.

Comment (McGiverGim) [reply azure-pipelines.yml:40]: I've made some tests in some online yaml validators, and it seems they accept both forms, so this seems ok...

Comment (mikeller) [reply azure-pipelines.yml:40]: Yeah, YAML is a lot like JS - unless it's obviously wrong it's assumed to be right.

**Pull request merged**